lib/utils/__tests__/addSemicolonForEditInfo.test.mjs

```diff
@@ -121,9 +128,10 @@ describe('addSemicolonForEditInfo', () => {
 
 			expect(fixData.text).toBe('@layer foo ');
 
-			const text = addSemicolonForEditInfo(node, fixData).text;
-
-			expect(text).toBe('@layer foo ');
+			expect(addSemicolonForEditInfo(node, fixData)).toEqual({
+				range: [0, 0],
+				text: '@layer foo;',
```

Comment:
question

`text` was `@layer foo ` it's now `@layer foo;`, is that on purpose?